spec/components/shared/filters_component_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Shared::FiltersComponent, type: :component do
+  class Form
+    def govuk_submit(arg1, arg2); end
+
+    def govuk_collection_check_boxes(arg1, arg2, arg3, arg4, arg5, arg6); end
+  end
+
+  let(:f) { Form.new }
+
+  context 'when there are selected filters' do
+    subject do
+      described_class.new(form: f, filters: { total_count: 2 }, items: [{ title: 'Group 1', key: 'group_1', options: [%w[option_1 OPTION_1]], selected: %w[option_1], value_method: :first, selected_method: :last }, { title: 'Group 2', key: 'group_2', options: [%w[option_1 OPTION_1]], selected: %w[option_1], value_method: :first, selected_method: :last }], options: { remove_buttons: true, close_all: true })
```

Comment:
Can we break this over several lines so it fits in the line limit please? Rubocop should pick this up 🤔

Reply:
no rubocop didnt obviously? but we are updating that config soon arent we?